Products/ZenModel/migrate/addDescriptionToCommands.py

```diff
@@ -0,0 +1,71 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+__doc__ = """
+Add a Description field to each Commands entry
+"""
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+
+
+class addDescriptionToCommands(Migrate.Step):
+
+    version = Migrate.Version(5, 1, 70)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        for service in ctx.services:
+            commands = service.commands
+            if not commands:
+                continue
+            for command in commands:
```

Comment:
Are we sure that all of the command names across all of the services are unique?  
Names like 'check' and 'initialize' sound kind of generic; i.e. 2 different services have different commands that happen to have the same name.